components/Counter/index.js

```diff
@@ -20,22 +25,22 @@ const StyledButton = styled.button`
 
 export default function Counter({ timerDelay }) {
   const [count, setCount] = useState(0);
-  const [intervalId, setIntervalId] = useState(null);
+
   const [isActive, setIsActive] = useState(false);
 
   const handleClick = () => {
-    const newIntervalId = setInterval(() => {
+    const breathInterval = setInterval(() => {
```

Comment:
The interval does not stop when the page is unmounted (you can test this by adding `console.log("tick")` to it), so you could run in a situation with several intervals being active simultaneously.
You can prevent this by storing the current `intervalId` in a state variable and adding this `useEffect()`:
```js
useEffect(()=>{
return ()=>{if(intervalId){clearInterval(intervalId)}}
},[])
```

Reply:
Thanks a lot for your comments, @actyralf. The button is already disabled while the counter is active (isActive state) to prevent starting several intervals at once. Also, the counter ist meant to stop on "40" without resetting or allowing any further user interaction. Another function ist going to be initialized automatically after one round of counting (in a future user story).  Do I still need the useEffect then?

Reply:
Try this: add a console.log to the interval callback. Then navigate to the homepage, then to your breathing page. Start the timer and navigate back home via browser back. The interval is still running. Navigate back to the breathing page, you can start a second interval then. You can see in the console that both intervals are running in parallel.